auth-server/docker-compose.yaml

```diff
@@ -12,4 +12,4 @@ services:
       PGDATA: /var/lib/postgresql/data
     volumes:
     # persistent data locally
-    - ./data:/var/lib/postgresql/data
+    - /var/polkassembly/auth-server/data:/var/lib/postgresql/data
```

Comment:
docker-compose.yaml can be removed completely. It is in gitignore now

Reply:
indeed, deleted it.